### PR TITLE
feat(components/list): disable sort on column header

### PR DESCRIPTION
**Pull request by lmaillet**
**What is the problem this PR is trying to solve?**
It's impossible to disable sorting on a column of a List.

**What is the chosen solution to this problem?**
Take into account the `disabled` prop of the `columns` of `List`.

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js [hunk 1]
```diff
@@ -92,6 +92,9 @@ export function ListToVirtualizedList(props) {
 							columnData: column.data,
 						});
 					}
+					if (column.disableSort) {
```
Comment (acateland) [inline packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js]: You can remove the condition altogether and assign column.disableSort to  cProps.disableSort

File: packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.test.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const props = {
 	items: [{ id: 3, label: 'my item', myactions: [{ foo: 'bar' }] }],
 	columns: [
 		{ key: 'id', label: 'Id', type: 'customType', header: 'customType' },
-		{ key: 'label', label: 'Label' },
+		{ key: 'label', label: 'Label', disableSort: true },
```
Comment (acateland) [inline packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.test.js]: Please add a new test to check that the disable option is applyed.

File: packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.test.js [hunk 3]
```diff
@@ -72,6 +72,20 @@ describe('ListToVirtualizedList', () => {
 		expect(table.rowHeight).toBe(200);
 	});
 
+	it('should support disableSort', () => {
```
Comment (acateland) [inline packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.test.js]: Please state in test description that headers that are hidden due to `hideHeader: true` should have the prop disableSort set to true to reflect your test code

File: packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js [hunk 4]
```diff
@@ -81,6 +81,7 @@ export function ListToVirtualizedList(props) {
 						label: column.label,
 						dataKey: column.key,
 					};
+					cProps.disableSort = column.disableSort;
```
Comment (jsomsanith) [inline packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js]: Please move that just 2 lines above in the `cProps` declaration

**Pull request merged**